src/builder.rs

```diff
@@ -423,19 +483,39 @@ impl ActionInfo {
     }
 
     /// Returns the value sum for this action.
+    /// Split notes do not contribute to the value sum.
```

Comment:
_From str4d ([2602410770](https://github.com/zcash/orchard/pull/471#discussion_r2602410770))_

```suggestion
    ///
    /// Split notes do not contribute to the value sum.
```